src/api_proxy/service_control/request_info.h

```diff
@@ -117,6 +117,10 @@ struct CheckResponseInfo {
   ScResponseErrorType error_type = ScResponseErrorType::ERROR_TYPE_UNSPECIFIED;
 
   std::string consumer_project_number;
```

Comment:
Is this still needed?

Reply:
Yes, we have a label still using consumer_project_number
https://github.com/GoogleCloudPlatform/esp-v2/blob/585f37109ee2cf831c97f279f0a3755572b32bb8/src/api_proxy/service_control/request_builder.cc#L595